packages/@okta/vuepress-site/docs/guides/terraform-create-custom-auth-server/index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Create a custom authorization server
```

Comment:
Actually you can both update and delete. To update, change values in the configuration and apply it. To delete it, remove the thing from the configuration and that should remove it from the org.

For the long title it makes sense as that's the focus of the article.

I used "Manage" in the navbar because although the article focusses on creation, managing is done the same way as other objects (update values, delete objects).

does it still need changing?

Reply:
No its fine.